### ISSUE TITLE
Java heap size too small

### ISSUE THREAD

**Issue by ndaniel** [ndaniel/fusioncatcher#138]
In some rare situations FusionCatcher v1.10 might fail when running Java tools, with an error message as:
```
java -ea -Xmx502m -cp /apps/fusioncatcher/tools/bbmap/ align2.BBMap build=1 overwrite=true fastareadlen=500 ...
Executing align2.BBMap [tipsearch=150, minhits=1, minratio=0.45, build=1, overwrite=true, fastareadlen=500 ...

Set MINIMUM_ALIGNMENT_SCORE_RATIO to 0.450
Set threads to 1
Retaining first best site only for ambiguous mappings.
Set genome to 1

Loaded Reference:	0.050 seconds.
Loading index for chunk 1-1, build 1
Generated Index:	1.624 seconds.
Exception in thread "main" java.lang.OutOfMemoryError: Java heap space

	at align2.BBIndex.analyzeIndex(BBIndex.java:108)

	at align2.BBMap.loadIndex(BBMap.java:418)

	at align2.BBMap.main(BBMap.java:33)
```

**Comment by ndaniel** [comment]
The heap size should be increased. This has been fixed in FusionCatcher 1.20.

**Comment by ndaniel** [comment]
Fixed in 1.20.
